lib/gui-utils/src/GUIUtils.cpp

```diff
@@ -106,17 +106,6 @@ void GUIUtils::displayBottomLine(String msg)
 #endif 
 }
 
-void GUIUtils::displayEndLine(String msg)
```

Comment:
What is the reason for delete this method? The end line will be not shown?

Reply:
Is not used now, is doing in other method